lib/src/defaults/providers/graphql/src/graphql_external_interface.dart

```diff
@@ -21,34 +20,46 @@ class GraphQLExternalInterface
   void handleRequest() {
     on<QueryGraphQLRequest>(
       (request, send) async {
-        try {
-          final data = await _graphQLService.request(
-            method: GraphQLMethod.query,
-            document: request.document,
-            variables: request.variables,
-          );
-          send(Right(GraphQLSuccessResponse(data: data)));
-        } catch (e) {
-          // log the details on the exception here
-          send(Left(FailureResponse()));
-        }
+        final data = await _graphQLService.request(
+          method: GraphQLMethod.query,
+          document: request.document,
+          variables: request.variables,
+        );
+
+        send(GraphQLSuccessResponse(data: data));
```

Comment:
Please let's not handle exceptions outside of External interface, that is one of the main design responsibilities for this class. We can discuss this more.